sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/setup.ps1

```diff
@@ -113,18 +113,36 @@ $appSecret = az ad app credential reset --id $appId --years 2 --query 'password'
 $user = $env:UserName
 $fileName = "$user.config.json"
 Write-Host("Writing user config file - $fileName`n")
-$appSecretJsonEscaped = ConvertTo-Json $appSecret
+
 $config = @"
 {
-    "DigitalTwinsInstanceHostName": "$dtHostName",
-    "ApplicationId": "$appId",
-    "ClientSecret": $appSecretJsonEscaped,
-    "TestMode":  "Live"
+    "TestMode":  "Live"	
 }
 "@
 
 $config | Out-File "$PSScriptRoot\..\config\$fileName"
 
+$outputfileDir = (Get-Item -Path $PSScriptRoot).Parent.Parent.Parent.Fullname
+$outputFile = Join-Path -Path $outputfileDir -ChildPath "test-resources.json.env"
+$tenantId = "72f988bf-86f1-41af-91ab-2d7cd011db47"
```

Comment:
If a customer ran this script to have infra to run tests, this hardcoded value would be a problem.